demo/src/main/java/com/orange/ods/demo/ui/components/chips/SubComponentChipFilter.kt

```diff
@@ -0,0 +1,102 @@
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.rememberBottomSheetScaffoldState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.google.accompanist.flowlayout.FlowRow
+import com.orange.ods.compose.component.chip.OdsFilterChip
+import com.orange.ods.demo.R
+import com.orange.ods.demo.ui.components.utilities.ComponentCustomizationBottomSheetScaffold
+import com.orange.ods.demo.ui.components.utilities.ComponentCustomizationCheckboxItem
+import com.orange.ods.demo.ui.utilities.composable.LabelledRadioButton
+import com.orange.ods.demo.ui.utilities.composable.Subtitle
+
+@ExperimentalMaterialApi
+@Composable
+fun SubComponentChipFilter() {
+    val customizationState = rememberChipCustomizationState()
+
+    ComponentCustomizationBottomSheetScaffold(
+        bottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
+        bottomSheetContent = {
+
+            Subtitle(textRes = R.string.component_element_leading, withHorizontalPadding = true)
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = dimensionResource(id = R.dimen.ods_spacing_xs))
+                    .padding(start = dimensionResource(id = R.dimen.ods_spacing_s)),
+                horizontalArrangement = Arrangement.Start
+            ) {
+                LabelledRadioButton(
+                    selectedRadio = customizationState.leadingElement,
+                    currentRadio = ChipCustomizationState.LeadingElement.None,
+                    label = stringResource(id = R.string.component_element_none)
+                )
+                LabelledRadioButton(
+                    selectedRadio = customizationState.leadingElement,
+                    currentRadio = ChipCustomizationState.LeadingElement.Avatar,
+                    label = stringResource(id = R.string.component_element_avatar)
+                )
+            }
+
+            ComponentCustomizationCheckboxItem(labelRes = R.string.component_state_outlined, checked = customizationState.outlinedChecked)
+            ComponentCustomizationCheckboxItem(labelRes = R.string.component_state_disabled, checked = customizationState.disabledChecked)
+
+        }) {
+        Column(
+            modifier = Modifier
+                .verticalScroll(rememberScrollState())
+                .padding(
+                    horizontal = dimensionResource(id = R.dimen.ods_screen_horizontal_margin),
+                    vertical = dimensionResource(id = R.dimen.ods_screen_vertical_margin)
+                )
+        ) {
+            FlowRow(modifier = Modifier.fillMaxWidth()) {
+                for (index in 1..4) {
```

Comment:
A detail, `repeat` is equivalent (I admit there is no real difference except that the intention is clearer):

```kotlin
repeat(4) { index ->
    //
}
```

Reply:
Ok thank you I do not know this function

Reply:
With the use of `repeat` the index starts to 0 so I have to use `index + 1` to display a correct number. What do you think?